### DISCUSSION TITLE
xrdp sound

### DISCUSSION THREAD

**Discussion by graylion** [neutrinolabs/xrdp#2922] [Q&A]
I have xrdp installed on my personal box and am rdping into it. Sound somehow doesn't make it. I see the dummy device and no amoit of starting pulseaudio seems to fix it.

I have worked through https://c-nergy.be/blog/?p=18148 but the modification to /tmp/ wasn't made since I installed xrdp from repo. 

Help?

**Comment by matt335672** [comment]
You need to log what the pulseaudio server is doing I think.

Start with :-

https://github.com/neutrinolabs/pulseaudio-module-xrdp/issues/101#issuecomment-1665369529

There's some more information in:-

https://github.com/neutrinolabs/pulseaudio-module-xrdp/issues/89
